challenges/stats_preview_card_component/index.html

```diff
@@ -25,39 +33,84 @@
       sizes="16x16"
       href="../images/favicon-16x16.png"
     />
-    <link rel="manifest" href="../site.webmanifest" />
-    <link rel="stylesheet" href="style.css" />
+
+    <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
+    <style>
+      .attribution {
+        font-size: 11px;
+        text-align: center;
+      }
+      .attribution a {
+        color: hsl(228, 45%, 44%);
+      }
+    </style>
   </head>
-  <body class="m-0">
+  <body class="bg-verydarkblue font-Inter">
     <div
-      class="flex h-[100vh] w-full flex-col place-content-center text-center"
+      class="container flex text-center sm:mr-auto sm:ml-auto sm:mt-10 sm:flex-col-reverse sm:p-5 lg:mt-44 lg:flex-row lg:gap-0"
```

Comment:
No need for `sm:` prefix as per [Tailwind guide](https://tailwindcss.com/docs/responsive-design#mobile-first) on mobile-first responsive design in this and other parts of the code.